apisix/http/router/radixtree_sni.lua

```diff
@@ -95,14 +109,22 @@ local function create_router(ssl_items)
             end
 
             -- decrypt private key
-            if aes_128_cbc_with_iv ~= nil and
-                not core.string.has_prefix(ssl.value.key, "---") then
-                local decrypted = aes_128_cbc_with_iv:decrypt(ngx_decode_base64(ssl.value.key))
-                if decrypted == nil then
-                    core.log.error("decrypt ssl key failed. key[", ssl.value.key, "] ")
-                else
+            if aes_128_cbc_with_iv ~= nil then
+                local decrypted = decrypt_priv_pkey(aes_128_cbc_with_iv,
```

Comment:
the `ssl.value.key` maybe nil. how about this code style?

```lua
if ssl.value.key then
   ...
end

if ssl.value.keys then
    ...
end
```

Reply:
sure.